extension-base/src/main/java/com/azure/autorest/extension/base/model/codemodel/Relations.java

```diff
@@ -3,23 +3,23 @@
 import java.util.List;
 
 public class Relations {
-    private List<ComplexSchema> all;
+    private List<Schema> all;
 
-    private List<ComplexSchema> immediate;
+    private List<Schema> immediate;
```

Comment:
It actually get a `AnySchema` in it, in this spec (though for me, that definition does not make much sense) https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cloudshell/resource-manager/Microsoft.Portal/stable/2018-10-01/CloudShell.json#L789-L809

Reply:
The change should not affect existing logic. For child class, it definitely still need to be `ObjectSchema`. For parent class, `AnySchema` will be ignored. Currently for parent class, `ObjectSchema` is good, `DictionarySchema` will be convert to additionalProperties.